tests/pass/shims/time-with-isolation.stdout

```diff
@@ -0,0 +1,2 @@
+The loop took around 7s
+(It's fine for this number to change a bit when you `--bless` this test.)
```

Comment:
How much is "a bit"? The recent rust-lang/rust PR changed this number from 7 to 12 which is quite a large swing normally, but per the comment in the source code within an order of magnitude is fine.

So probably we should say "within an order of magnitude" here if that is our expectation.

Reply:
We can also just say "it is okay to change"; we'll get pinged anyway when it happens so we can intervene if something seems to go terribly wrong.

Reply:
Works for me!